code/io/importYaml.m

```diff
@@ -25,6 +25,25 @@
     error('Yaml file %s cannot be found', string(yamlFilename));
 end
 
+if verLessThan('matlab','9.9') %readlines introduced 2020b
+    fid=fopen(yamlFile);
```

Comment:
A bit confusing with yamlFile and yamlFilename - it looks like yamlFilename is the right one, if I look at line 1?